manager/integration/tests/test_recurring_job.py

```diff
@@ -483,3 +483,41 @@ def test_recurring_jobs_for_detached_volume():  # NOQA
     17. cleanup
     """
     pass
+
+
+@pytest.mark.skip(reason="TODO")
+def test_recurring_jobs_on_nodes_with_taints():  # NOQA
+    """
+    Test recurring jobs on nodes with taints
+
+    Context:
+
+    longhorn/longhorn#1903
```

Comment:
Can we write something like 'Test the prevention of the creation of multiple pods due to recurring job failure' instead of the bug id?

Reply:
Thank you! Fixed